### DISCUSSION TITLE
The drawing board corner - naptime when?

### DISCUSSION THREAD

**Discussion by PatribotsProgramming** [Patribots4738/Crescendo2024#50] [Show and tell]
This discussion will contain all of the drawing boards used in this season to keep in one place 😄

Below are the final versions :D please feel free to explore! (the comments are just the culminations of everything)
### View only edition: [here](https://www.tldraw.com/v/YKJloESPqAyu62wxqEQ8U?v=1783,102,6548,3115&p=page)

**Comment by jacueblol** [comment]
I was just abt to make this lol

**Reply by PatribotsProgramming** [reply]
teehee

**Comment by PatribotsProgramming** [comment]
I wanted to post this here since it has nice colors and I want it on all the cool computers
![patri-alliancec](https://github.com/Patribots4738/Crescendo2024/assets/97857553/a036572b-dbed-45ce-908d-da6d59f6c643)

**Comment by rin-star** [comment]
Link for READ OR VIEW ONLY:

https://www.tldraw.com/v/YKJloESPqAyu62wxqEQ8U?viewport=0,0,1536,695&page=page:9NTiPVa29oqzjElya5D6n
